common/src/main/cpp/register_types.cpp

```diff
@@ -96,8 +101,10 @@ void initialize_plugin_module(ModuleInitializationLevel p_level) {
 			Engine::get_singleton()->register_singleton("OpenXRFbSpatialEntityContainerExtensionWrapper", OpenXRFbSpatialEntityContainerExtensionWrapper::get_singleton());
 			Engine::get_singleton()->register_singleton("OpenXRFbSceneExtensionWrapper", OpenXRFbSceneExtensionWrapper::get_singleton());
 			Engine::get_singleton()->register_singleton("OpenXRFbFaceTrackingExtensionWrapper", OpenXRFbFaceTrackingExtensionWrapper::get_singleton());
+			Engine::get_singleton()->register_singleton("OpenXRFbHandTrackingMeshExtensionWrapper", OpenXRFbHandTrackingMeshExtensionWrapper::get_singleton());
```

Comment:
You can remove this registration since the api is exposed via the mesh node instead.

Reply:
Removed this as well as the `is_enabled` function binding in latest push.